### ISSUE TITLE
Support Swift Package Manager in Xcode 11

### ISSUE THREAD

**Issue by optikfluffel** [davidstump/SwiftPhoenixClient#154]
It would be really nice to be able to use the Swift Package Manager, since it's built right into Xcode now.

**Comment by dsrees** [comment]
You should already be able to

```
.package(url: "git@github.com:davidstump/SwiftPhoenixClient.git", from: "1.1.0"),
```

**Comment by optikfluffel** [comment]
I can add it via Xcode:

<img width="1392" alt="Screenshot 2019-12-18 at 22 55 54" src="https://user-images.githubusercontent.com/379915/71126681-c423d280-21e9-11ea-8844-d71b2c811de4.png">

But as soon as I try to import it anywhere, I just get `No such module 'SwiftPhoenixClient'`:

![Screenshot 2019-12-18 at 22 51 20](https://user-images.githubusercontent.com/379915/71126496-65f6ef80-21e9-11ea-902f-e6dce9eda4ca.png)

Am I missing anything?

**Comment by dsrees** [comment]
Hmm, looks like xcode isn't recognizing the library. I'll have to look more into this.

**Comment by dsrees** [comment]
Fixed in 1.2.0

**Comment by optikfluffel** [comment]
Thank you so much!
